src/turbine/templates/python/main.py

```diff
@@ -8,29 +9,32 @@
 
```

Comment:
Moving this to a module level rather than a method level will setup the logger once. 
```suggestion
logging.basicConfig(level=logging.INFO)

```